src/Host.h

```diff
@@ -56,16 +56,13 @@ class dlgNotepad;
 class TMap;
 
 
-class Host  : public QObject
-{
+class Host : public QObject {
     friend class XMLexport;
     friend class XMLimport;
 
 public:
-
-                       Host( int port, const QString& mHostName, const QString& login, const QString& pass, int host_id );
-
-                       ~Host();
+    Host(int port, const QString& mHostName, const QString& login, const QString& pass, int host_id);
```

Comment:
It's hard to know if what you want is possible without knowing what you want it to look like. One of the longer term issues is that I'd like to have doc comments on all functions but those would prevent clang-format from being able to align the names since they'd no longer exist on adjacent lines. I also don't like aligning them when they are on adjacent lines, but that's irrelevant if aligning names conflicts with adding docs in the future.